rigs/kenwood/ic10.c

```diff
@@ -152,7 +152,7 @@ static int get_ic10_if(RIG *rig, char *data)
 {
     const struct kenwood_priv_caps *priv = (struct kenwood_priv_caps *)
                                            rig->caps->priv;
-    int i, data_len, retval = RIG_EINVAL;
```

Comment:
This is consistent with the other return value of this function which is `-RIG_ERJCTED`.